crowbar_framework/app/models/neutron_service.rb

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+require_relative '../helpers/barclamp/neutron_helper.rb'
```

Comment:
@tboerger I need a list of valid ml2 drivers in the helpers (for the ```options_for_select()``` method) **and** the model (for validation). To which file should I add this list?